aleph-im/aleph-account
# New version scoring

- feat: Display node scores on the staking UI
- chore: remove hardcoded uptime column
- ref: (minor) avoid unecessary computations
- fix: scoring
- ref: Nodes with 0 score set to 0% instead of n/a
- enh: spacing
- enh: (ux) display score as bars (#66)
- enh: crn and ccn do not have the same metrics
- fix: register shouldn't be clickable if not logged
- enh: tooltip if not clickable
- chore: remove solana connect button (UI)
- fix: resource node don't have a registration url
- feat: display customized score
- rm: useless console.log
- feat: display outdated node warning
- enh: tooltip on node version
- chore: rm frontend total_score computation
- enh: use hashmap based join
- fix: only display outdated version for CCN
- fix: lint
- fix: wrong key check
- chore: edit scoring display
- add: CCN new scoring
- add: CRN display (wip)
- add: CRN scoring
- fix: weird display bug
- fix: quality column based on "performance" field
- fix: Unreachable nodes are obsolete by default
- chore: rm "test-" msg prefix
- ref: version check
- add: grace period warning
- tmp: experimental node displayed in orange
- fix: can't filter crn on name/rewards
- fix: grace period warning is now responsive
- add: tooltip on state (base)
- enh: add link to medium in disclaimer
- enh: tooltip
- enh: score tooltip
- enh: Add CRN penalty display
- Temp: Use test address
